com.chartboost.mediation.demo/Assets/Demo/Demo.cs

```diff
@@ -69,8 +110,8 @@ private void OnDestroy()
         if (_bannerAd == null) 
             return;
         
-        _bannerAd.ClearLoaded();
-        _bannerAd.Destroy();
+        _bannerAd.ResetAd();
+        Destroy(_bannerAd);
```

Comment:
Is this Destroy implementation necessary? If so, wouldn't it be better for it to be part of the `ChartboostMediationUnityBannerAd` itself? `ChartboostMediationUnityBannerAd` is a `MonoBehavior` so it should have everything it needs to implement it on the class.

Reply:
Also, this destroy is only destroying the `MonoBehavior` not the gameobject that contains the `MonoBehavior`, that does not seem correct.

Reply:
yeah it should be `Destroy(_bannerAd.gameObject)` I have it fixed in another PR that will be coming next but I'll add that fix here too in next commit

`ChartboostMediationUnityBannerAd` implements `OnDestroy` of the monobehaviour which destroys the underlying`BannerView` on native. In my opinion, for a unity developer the natural way to destroy a monobehaviour is by calling Destroy(<monobehaviour>)  so it didn't feel necessary to add a separate Destroy call 

That's just my thought though, if you feel adding a separate method would be better then I'll be more than happy to add that

Reply:
Nvm I thought the destroying part you were talking about is in `OnBannerRemove` didn't notice it was from `OnDestroy` of demo. You are right, there's no need to call destroy there since `OnDestory` on `ChartboostMediatinoUnityBannerAd` will take care of it